npm/vue/cypress.config.js

```diff
@@ -0,0 +1,32 @@
+const { startDevServer } = require('@cypress/webpack-dev-server')
+
+module.exports = {
+  viewportWidth: 500,
+  viewportHeight: 500,
+  video: false,
+  responseTimeout: 2500,
+  projectId: '134ej7',
+  testFiles: '**/*spec.js',
+  experimentalFetchPolyfill: true,
+  component: (on, config) => {
```

Comment:
If we want to customize some values for component, eg viewport, what is the recommendation?

## Before

```json
{
  "component": {
    "viewportHeight": 400
  }
}
```

## After

```js
module.exports = {
  component: (on, config) => {
    // ...
    return {
      ...config,
      viewportHeight: 400
    }
  }
}
```

Is this correct? For anyone with overrides, the migration won't be as simple as rename + wrap in `module.exports`. We might want to include this in the PR description and (eventual) docs.

Reply:
I think this is how it was intended indeed.

This or a simple affectation (no need for the spread)

```js
module.exports = {
  component: (on, config) => {
    // ...
	config.viewportHeight = 400
    return config
  }
}
```